### ISSUE TITLE
glGetInteger throwing GL_INVALID_ENUM

### ISSUE THREAD

**Issue by therealergo** [LWJGL/lwjgl3#84]
Don't know if it's just me making a mistake or just my machine, but with LWJGL 3.0.0 GL11.glGetInteger(GL20.GL_MAX_COMBINED_TEXTURE_IMAGE_UNITS) seems to trigger a GL_INVALID_ENUM error when called, despite that pair being used fairly often to query the number of available texture units.

**Comment by Spasi** [comment]
Cannot reproduce. Could you please try it in a debug context and post the error output here?

```java
glfwWindowHint(GLFW_OPENGL_DEBUG_CONTEXT, GL_TRUE);
// ... (create window) ...
glfwMakeContextCurrent(window);
GL.createCapabilities();
debugProc = GLUtil.setupDebugMessageCallback();
```

**Comment by therealergo** [comment]
Updated to the latest nightly build, debug output is:
(Sun Sep 06 09:57:29 CDT 2015)(Render Thread)> [LWJGL] OpenGL debug message
(Sun Sep 06 09:57:29 CDT 2015)(Render Thread)> 	ID: 0xC80
(Sun Sep 06 09:57:29 CDT 2015)(Render Thread)> 	Source: API
(Sun Sep 06 09:57:29 CDT 2015)(Render Thread)> 	Type: ERROR
(Sun Sep 06 09:57:29 CDT 2015)(Render Thread)> 	Severity: HIGH
(Sun Sep 06 09:57:29 CDT 2015)(Render Thread)> 	Message: Using glGetIntegerv in a Core context with parameter <pname> and enum '0x8871' which was removed from Core OpenGL (GL_INVALID_ENUM)
    A forward-compatible 3.2 core context is being used, and the error can be stopped either by allowing a backwards-compatible context or by calling GL.createCapabilities() at any point before the error in versions before 3.0.0b build 9.
    It seems that GL_MAX_COMBINED_TEXTURE_IMAGE_UNITS is disabled in 3.2+, is there any way of me verifying this? Googling seems to reveal nothing on the subject, and I don't see how I'd get the maximum number of texture units correctly without it.

**Comment by Spasi** [comment]
`0x8871` is `MAX_TEXTURE_COORDS`, not `MAX_COMBINED_TEXTURE_IMAGE_UNITS`.

**Comment by sriharshachilakapati** [comment]
Are you using value instead of the constant name? `MAX_COMBINED_TEXTURE_IMAGE_UNITS` is equal to `0x8B4D` when I printed it out.

**Comment by therealergo** [comment]
Well, I've at least figured out what it was -- My Eclipse now only saves files from the project that is being run when that project is run, rather than saving all open files -- Which means that because I run more than 1 project with referenced source directories between them when I deleted GL_MAX_TEXTURE_COORDS  it didn't save and kept triggering the error, leading me to believe that it was GL_MAX_COMBINED_TEXTURE_IMAGE_UNITS instead of GL_MAX_TEXTURE_COORDS. It makes sense now that GL_MAX_TEXTURE_COORDS was removed because it is for the fixed pipeline and should no longer be used. Anyways, thanks for all the help, I'm glad I got this dealt with.
